README.md

```diff
@@ -22,7 +22,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 |9|Meeting Recruitment App | Sample app showing meeting app experience for interview scenario. |[View](/demo%20manifest/Meeting-Recruitment-App.zip)     |
 |10|App installation using QR code | Sample app that generates a QR or installs app using the QR code.|[View](/demo%20manifest/App-Installation-Using-QR.zip)     |
 |11|App checkin location | Demonstrating feature where user can checkin with current location and view all previous checkins.|[View](/demo%20manifest/App-checkin-location.zip)     |
-
+|11|Connector Todo Notification | This is a sample app which shows connector authentication and sends notification on task creation.|    |[View](/demo%20manifest/connector-todo-notification.zip) 
```

Comment:
We already have Connector sample deployed. It's number 4 in this list. We can remove this as it's a duplicate scenario.